apps/marketplace/actions/buyerDashboardActions.js

```diff
@@ -0,0 +1,6 @@
+import dmapi from '../services/apiClient'
+
+const loadBuyerDashboard = status => () =>
+  dmapi({ url: `/buyer/dashboard`, params: { status } }).then(response => response)
```

Comment:
Do we need `.then(response => response)`?

Reply:
it looks cool? removed...